conjure-java-client-verifier/src/test/resources/ignored-test-cases.yml

```diff
@@ -65,13 +66,11 @@ client:
    - 'null'
    receiveSetAnyAliasExample:
    - 'null'
+   - '[null, 0, "content", true, [1,2,3], {"key":3}]' # null is not a valid any value, see conjure-verification#141
 
    receiveMapBinaryAliasExample:
    - '{}'
    - '{"SGVsbG8sIFdvcmxk": true}'
-   receiveMapDoubleAliasExample:
-   - '{"10": true, "10e0": false}'
-   - '{"10": true, "10.0": false}'
```

Comment:
Sadly, I don't think we can make a change like this without a major rev of conjure-java-runtime (which has historically been _super_ expensive). I think people will get super angry if we impose this change on them, because if they have odd looking JSON already persisted in their database they would suddenly be read it out again.

I think the way to make progress in this direction would be to allow users to opt-in to stricter inner fields in our builders:

```diff
   @Generated("com.palantir.conjure.java.types.BeanBuilderGenerator")
    @JsonIgnoreProperties(ignoreUnknown = true)
    public static final class Builder {
-       private List<String> value = new ArrayList<>();
+       private ImmutableList.Builder<String> value = ImmutableList.builder();

        private Builder() {}
```

This itself is a bit contentious, as we have previously tried to make sure our conjure objects don't rely on guava at all.  Maybe long-term we could let people use the JDK-native `List` https://docs.oracle.com/javase/9/docs/api/java/util/List.html#immutable?